registry.json

```diff
@@ -1937,7 +1938,8 @@
       "source": "https://github.com/binance-chain/bsc",
       "rpc": "https://bsc-dataseed1.binance.org",
       "documentation": "https://eth.wiki/json-rpc/API"
-    }
+    },
+    "testFolderName" : "Binance"
```

Comment:
There are 2 coins with testFolderName=Binance, will they not overwrite?

Reply:
Those 2 blockchains actually use one folder for their tests. It will not overwrite because the script will ignore the folders that already exist